src/components/grid/__snapshots__/grid.spec.js.snap

```diff
@@ -220,7 +220,7 @@ exports[`Grid GridItem builds the style rules for the GridItem responsiveSetting
           "attrs": Array [],
           "componentStyle": ComponentStyle {
             "componentId": "sc-bdVaJa",
-            "isStatic": true,
+            "isStatic": false,
```

Comment:
that's weird, we need to investigate that

Reply:
looks like we have a badly done snapshot, I'm resolving that one

Reply:
Will fix the two instances of bad snapshots (with internal styled-components) that have been highlighted by this update in a separate PR